packages/soci-snapshotter/soci-snapshotter.spec

```
@@ -0,0 +1,53 @@
+%global gorepo soci-snapshotter
```

To verify that this builds correctly with FIPS and non-FIPS binaries, after rebasing on develop:
```suggestion
%global cross_check_fips 1

%global gorepo soci-snapshotter
```